Rakefile

```diff
@@ -1,10 +1,10 @@
 # Add your own tasks in files placed in lib/tasks ending in .rake,
 # for example lib/tasks/capistrano.rake, and they will automatically be available to Rake.
 
-require_relative 'config/application'
+require_relative "config/application"
 
 Rails.application.load_tasks
 
-task lint_ruby: ['lint:ruby']
-task lint_scss: ['lint:scss']
-task default: %i[spec lint_ruby lint_scss]
+if %w[development test].include?(Rails.env)
+  task default: ["lint:ruby", "lint:scss", :spec]
+end
```

Comment:
Why remove these tasks? Al's changed the build workflow to rely on these existing.

Also, just to note, there are changes in this file that'll conflict when you rebase onto master.

Reply:
I haven't removed them they're still in lib. They were just re-defined under the `task` command so they could be referenced using the `%i` syntax

Reply:
My bad. Good tidy up.